src/csharp/Microsoft.Spark/Sql/ArrowUdfWrapper.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Spark.Sql
     /// <typeparam name="T">Specifies the type of the first argument to the UDF.</typeparam>
     /// <typeparam name="TResult">Specifies the return type of the UDF.</typeparam>
     [UdfWrapper]
-    internal sealed class ArrowUdfWrapper<T, TResult>
+    internal sealed class ArrowUdfWrapper<T, TResult> 
```

Comment:
It looks like this is all unnecessary white space changes. This whole file can be reverted.